lib/Db/ItemMapper.php

```diff
@@ -50,10 +50,10 @@ public function __construct(IDBConnection $db, Time $time)
     }
 
     private function makeSelectQuery(
-        $prependTo = '',
+        string $prependTo = '',
         $oldestFirst = false,
         $distinctFingerprint = false
```

Comment:
these two could probably get the type `bool`

Reply:
Added the bool, the second one is not even used though and later in the code when this function gets called a $search is passed... but let's not look at that too much for now, it might break if we do.
https://github.com/Grotax/news/blob/ec234cfdbcab00094221ba7b7a778a2ef5d81d50/lib/Db/ItemMapper.php#L312